DPG/Tasks/AOTTrack/qaMatchEff.cxx

```diff
@@ -120,6 +123,22 @@ struct qaMatchEff {
     Configurable<float> etaMaxCut{"etaMaxCut", 2.0f, "Maximum pseudorapidity"};
   } kineCuts;
   //
+  //
+  AxisSpec axisPDG{pdgBins, 0, pdgBins + 1.000, "pdgclass"};
+  //
+  AxisSpec axisPt{ptBins, "#it{p}_{T} (GeV/#it{c})"};
+  //
+  AxisSpec axisX{XBins, "track x (cm)"};
+  //
+  AxisSpec axisZ{ZBins, "track z (cm)"};
+  //
+  AxisSpec axisQoPt{qoptBins, -20, 20, "#Q/it{p}_{T} (GeV/#it{c})^{-1}"};
+  //
+  AxisSpec axisEta{etaBins, kineCuts.etaMinCut, kineCuts.etaMaxCut, "#eta"};
+  AxisSpec axisPhi{phiBins, 0.f, TwoPI, "#it{#varphi} (rad)"};
+  AxisSpec axisDEta{etaBins, kineCuts.etaMinCut, kineCuts.etaMaxCut, "D#eta"};
+  AxisSpec axisDPh{phiBins, -PI, PI, "D#it{#varphi} (rad)"};
```

Comment:
@rosarioturrisi maybe it's better to put also these axes in the `init` functions, for the same reasons

Reply:
Thanks for spotting them @mfaggin, just modified the task.